.babelrc

```diff
@@ -1,7 +0,0 @@
-{
```

Comment:
This was the single place for babel config which is implicitly used by webpack, mocha for front-end and eslint